instfiles/Makefile.am

```diff
@@ -50,7 +50,9 @@ dist_startscript_DATA = \
   km-00000813.ini \
   km-00000816.ini \
   km-0000100c.ini \
-  km-00010409.ini
+  km-00010409.ini \
+  km-19360409.ini \
```

Comment:
Backslash must not be put at the end.

Reply:
It was actually intentional to have a blank line at the end, to avoid having every change overwrite the blame of the previous line to add the backslash, but I'm not invested in it, so it's changed now.

Reply:
Thanks, it actually caused a build failure.